### PR TITLE
Invalid legit outcome

### LINKED ISSUE #458: Make Invalid outcome explicit
In v2 there is no longer a special reporting flag to indicate a market is Invalid.

Instead the first outcome for every market is the Invalid outcome. It can be traded just like other outcomes and when reporting a market as Invalid one must place the entire payout onto the Invalid outcome.

### PR DESCRIPTION
**Pull request by bthaile**
git flash scripts working with initial report and dispute.

updated claim-reporting.... source files and fixed augur.js tests 

closes https://github.com/AugurProject/augur/issues/458

### CODE REVIEW

File: packages/augur-tools/scripts/flash/index.js [hunk 1]
```diff
@@ -115,13 +116,9 @@ var methods = {
...
         help:
-          'Outcome, sets outcome to use, can be overridden by invalid flag, negative outcome use \\"-10\\"'
-      },
-      invalid: {
-        flag: true,
-        short: "i",
-        help: "Overrides outcome to pass invalid"
+          'Outcome, sets outcome to use, negative outcome use \\"-10\\"'
       },
+      description: { default: "", short: "d", help: "Add description" },
```
Comment (justinbarry) [inline packages/augur-tools/scripts/flash/index.js:121]: What is an "invalid" description?

Comment (justinbarry) [reply packages/augur-tools/scripts/flash/index.js:121]: Should it be aliased to something like outcome id?

File: packages/augur-tools/scripts/flash/get-payout-numerators.js [hunk 2]
```diff
@@ -21,22 +21,26 @@ function getPayoutNumerators(market, selectedOutcome, invalid) {
   var numTicks = createBigNumber(market.numTicks);
   var numOutcomes = market.numOutcomes;
 
+  console.log("numOutcomes", numOutcomes);
+  console.log("numTicks", numTicks);
+  console.log("selectedOutcome", selectedOutcome);
+  console.log("maxPrice", maxPrice.toFixed());
+  console.log("minPrice", minPrice.toFixed());
```
Comment (bthaile) [inline packages/augur-tools/scripts/flash/get-payout-numerators.js:28]: These console.logs are removed in my continuous dispute PR.

File: packages/augur-tools/scripts/flash/index.js (hunk 1 above)
Comment (bthaile) [reply packages/augur-tools/scripts/flash/index.js:121]: new string param on contracts. I default the value in next PR.

**Review by justinbarry — APPROVED** [review]
LGTM

**Pull request merged**